File: dev-docs/snippets/documents/add-tag.swift
import JsBaoClient

// Add a tag to a document. Returns the updated tag list as `[String]`.
func addTag(client: JsBaoClient, documentId: String) async throws {
  // #region example
  let tags = try await client.documents.addTag(documentId: documentId, tag: "archived")
  // #endregion example
  _ = tags
}

File: dev-docs/snippets/documents/add-tag.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Add a tag to a document. Returns the updated tag list.
export async function addTag(client: JsBaoClient, documentId: string) {
  // #region example
  const tags = await client.documents.addTag(documentId, "archived");
  // #endregion example
  return tags;
}
